packages/popover2/src/contextMenu2.tsx

```diff
@@ -129,6 +130,9 @@ export const ContextMenu2: React.FC<ContextMenu2Props> = React.forwardRef<any, C
         ...restProps
     } = props;
 
+    // ancestor Tooltip2Context state doesn't affect us since we don't care about parent ContextMenu2s, we only want to
+    // force disable parent Tooltip2s in certain cases through dispatching actions
+    const [, popover2Dispatch] = React.useContext(Tooltip2Context);
```

Comment:
```suggestion
    const [, tooltip2Dispatch] = React.useContext(Tooltip2Context);
```

should be renamed here and below...